flink-table/flink-table-planner/pom.xml

```diff
@@ -285,6 +340,8 @@ under the License.
 									<include>com.google.guava:guava</include>
 									<include>net.hydromatic:*</include>
 									<include>com.esri.geometry:*</include>
+									<include>com.fasterxml.jackson.*:*</include>
```

Comment:
I was following the last Calcite upgrade to include here this here, but I dont think this is needed anymore since it is specifically included in the `<dependencies/>`